tests/test_contrib.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016-2020 CERN.
+#
+# Marshmallow-Utils is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test the marshmallow contrib."""
+
+import pytest
+from marshmallow import Schema, ValidationError, fields
```

Comment:
can be removed since not used.

```suggestion
from marshmallow import Schema, ValidationError
```